data_science/terraform/variables.tf

```diff
@@ -1,15 +1,13 @@
+variable "key_name" {}
+
 variable "aws_region" {
   default = "eu-west-1"
 }
 
-variable "hashed_password" {
-  # The default password is 'password'
-  # To generate a new password run the following Python code:
-  #
-  # from notebook.auth import passwd
-  #   passwd()
-  #
-  default = "sha1:5310f21e370d:a4d66e725c179218638c21c03d83933aa066db2d"
+variable "vpc_cidr_block" {
+  default = "18.0.0.0/16"
 }
```

Comment:
I know it's possible, but perhaps we should be a bit careful about using 18.0.0.0/16?

Reply:
What issues might there be with using that block?